Src/BinaryConverter.Tests/BinaryConverter.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
```

Comment:
Why not NUnit or xUnit? MSTest has the least set of features compared to others.

Regardless of testing framework, you also can use a separate assertion library like [Shouldly](https://www.nuget.org/packages/Shouldly) or [FluentAssertions](https://www.nuget.org/packages/FluentAssertions/).